content/app-dev/create-provider/new-interface.md

```diff
@@ -81,8 +81,13 @@ wash new interface payments
 ```
 
 This command creates a new interface project in the subdirectory 'payments'. 
-Wasmcloud interfaces are defined in the [smithy](https://awslabs.github.io/smithy/) IDL. 
-When changes are made to the `.smithy` file, a code generator generates code for the selected target languages - currently only Rust is supported, and more languages will be supported in the future. The `make` command runs the code generator to generate the rust library, and compiles it. The generated library defines function signatures and data structures that will be shared by the Payments capability provider and the actor that calls the provider.
+Wasmcloud interfaces are defined in the [smithy](https://awslabs.github.io/smithy/) IDL. When changes are made to the `.smithy` file, a code generator generates code for the selected target languages - currently only Rust is supported, and more languages will be supported in the future. 
+
+The `wash new interface` command will ask for some information in order to fill in some meta-data in the generated code. First it will ask if you want to generate a `converter-interface` or a `factorial-interface`. The former is for an interface between actors so you would want to choose `factorial-interface` which is between actor and a capability provider. Use arrow-keys to select the right one.
+
+Next you need to give the interface a name. Use `wasmcloud-examples-payments`here. Namespec prefix should for this example be: `org.wasmcloud.examples`. The capability contract name should be `wasmcloud:examples:payments`. The final thing to do is to change the name of the package in the generated file `rustt/Cargo.toml` to `wasmcloud-examples-payment`. 
```

Comment:
Before giving the interface name, you have to select a project name. Let's suggest `payments` here first

Reply:
Typo in `Namespec prefix`, should be `Namespace prefix`

Reply:
Last thing, `rustt/Cargo.toml` has an extra `t` in there

Reply:
Thanks, however, after choosing project template, the next question is about Interface name. Maybe you are on a more advanced wash?

Reply:
Sorry for the misspellings.

Reply:
Are you on `0.7.2`? That's the version that I'm on.

Reply:
yes

Here is my output from the generator:
```
mats@gothmog:~/wasmcloud/xxx$ wash new interface payments
✔ Select a project template: · factorial-interface: an interface for a capability provider with capability contract
🔧   Using template subfolder `interface/factorial`...
🤷   Interface name (a short identifier such as 'messaging' or 'search')? [default: factorial]: wasmcloud-examples-payments
🤷   Namespace prefix? [default: org.example.interfaces]: org.wasmcloud.examples
🤷   Capability contract name? [default: example:interfaces:wasmcloud_examples_payments] [example:interfaces:wasmcloud_examples_payments]: wasmcloud:examples:pa🤷   Capability contract name? [default: example:interfaces:wasmcloud_examples_payments]: wasmcloud:examples:payments
🔧   Generating template ...
[ 1/16]   Done: .gitignore
[ 2/16]   Done: Makefile
[ 3/16]   Done: README.md
[ 4/16]   Done: codegen.toml
[ 5/16]   Done: wasmcloud_examples_payments.smithy
[ 6/16]   Done: html/org_wasmcloud_core.html
[ 7/16]   Done: html/org_wasmcloud_interface_factorial.html
[ 8/16]   Done: html/org_wasmcloud_model.html
[10/16]   Done: interface.mk
[11/16]   Skipped: project-generate.toml
[12/16]   Done: rust/Cargo.toml
[13/16]   Done: rust/build.rs
[14/16]   Done: rust/src/lib.rs
✨   Done! New project created /home/mats/wasmcloud/xxx/payments
```

Reply:
Must just be on my end 👍🏻 It's not a big deal either way, project name doesn't matter 🙂